e2e-tests/web/wdio.conf.js

```diff
@@ -0,0 +1,33 @@
+exports.config = {
+  runner: 'local',
+  specs: [
+    './web/test/specs/**/*.ts'
+  ],
+  exclude: [],
+  maxInstances: 10,
+
+  capabilities: [{
+    maxInstances: 5,
+    browserName: 'chrome',
+    acceptInsecureCerts: true
+  }],
+  logLevel: 'info',
+  coloredLogs: true,
+  bail: 0,
+  baseUrl: 'http://localhost:9000',
+  waitforTimeout: 100000,
+  connectionRetryTimeout: 120000,
+  connectionRetryCount: 3,
+  services: ['selenium-standalone'],
+  framework: 'jasmine',
+  reporters: ['junit'],
+
+  jasmineNodeOpts: {
+    defaultTimeoutInterval: 120000
+  },
+
+  onPrepare: async function () {
+    const startupDelay = 20000
+    await new Promise(resolve => setTimeout(resolve, startupDelay))
```

Comment:
Are we always waiting 20s before running tests? If yes, why?

Reply:
To wait for the webpack-server, as yarn start is executed beforehand. As the jasmine timeout is quite large it might be unnecessary, but I think this way is more correct.